.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -9,6 +9,11 @@ template_path=.github/ISSUE_TEMPLATE/feature_request.md
 -->
 ### [READ] Guidelines
 
+Consider instead starting a
+[Pitch](https://github.com/firebase/firebase-ios-sdk/discussions/categories/pitches)
+discussion for ideas that you'd like to discuss with the Firebase community to
+flush out.
+
```

Comment:
We should add some criteria here that disambiguates whether or not users should file a pitch or feature request. Maybe something like "For large or ambiguous requests, such as significant breaking changes or use cases that could be addressed by multiple different features, consider ..."

Reply:
Done Thanks.